packages/lism-ui/src/components/Avatar/_style.css

```
@@ -1,9 +1,22 @@
 @layer lism-block {
-  /* Note: l--frame との併用を前提としています. */
+  /* Note: src ありは l--frame、イニシャル表示は l--center との併用を前提としています. */
   .b--avatar {
     --w: 2em; /* size prop未指定時のデフォルトサイズ */
     width: var(--w);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)